fatek/symbol.py

```diff
@@ -74,7 +74,8 @@ def __calculate_offset(self):
         offset = self.offset_dict[register]
         if register == 'C' and current_value and number >= 200:
             number %= 200
-            offset = 9700 + 2 * (number)
+            number = number + number
+            offset = 9700
```

Comment:
but wait, for C255, with `current_value` it will be :
```
> number == 255
number %= 200
> number == 55
offset = 9700 + 2 * 55
> offset == 9810
```
its those the numbers ?